repository-commons/src/main/java/io/aiven/elasticsearch/repositories/AbstractRepositoryPlugin.java

```diff
@@ -43,6 +45,10 @@
 
     private final Set<String> pluginSettingKeys;
 
+    static {
```

Comment:
Just out of curiosity. I think it is possible to ass security provider statically, using java.security file,right? Why don't we did that?

Reply:
Let's do it explicitly